pipelines/test/virtualpackage.yaml

```diff
@@ -41,3 +41,24 @@ pipeline:
           exit 1
         fi
       fi
+
+  - name: virtual package configuration checks
+    description: Check that a a real package declares it provides the expected virtual package
+    runs: |
+      virtual_pkg=${{inputs.virtual-pkg-name}}
+      real_pkg=${{inputs.real-pkg-name}}
+
+      if [ "${real_pkg}" == "" ] ; then
+        echo "SKIP: no 'real-pkg-name' was set, so skipping evaluation."
+        exit 0
+      fi
+
+      declared_provides=$(apk info --installed --provides $real_pkg | tail -n +2 | grep -v ^$)
```

Comment:
does `--quiet` help you here? then you can drop the tail -n+2 i think.

you should also quote "$real_pkg" there.